### ISSUE TITLE
Sampling, Accuracy, and Time

### ISSUE THREAD

**Issue by wangziyue-nubot** [raktimgg/FlashMix#7]
Great work. But I have some questions.
As described in section 4.1 of the paper, for training the method for the Oxford-Radar dataset, authors sample point clouds every 1 meter. This will reduce the number of training samples, which is beneficial for fast training in situations where GPU performance is limited, but it will also lead to a decrease in localization accuracy. May I ask if the author trained on devices with larger memory to obtain the relationship between sampling ratios, training time, and positioning accuracy.

**Comment by raktimgg** [comment]
We did not train on larger GPUs to explore the relationship between sampling ratios, training time, and accuracy. Your observation is correct that training on more samples could increase localization accuracy. However, since the encoder is pre-trained on a large dataset, the performance difference should not be very high.
